site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,52 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+You can't migrate Agones game servers between Standard and Autopilot clusters.
+
+#### Agones on GKE Autopilot
+
+Autopilot is GKE's fully-managed mode. GKE configures, maintains, scales, and
+upgrades nodes for you, which can reduce your maintenance and operating
+overhead. You only pay for the resources requested by your running Pods, and
+you don't pay for unused node capacity or Kubernetes system workloads.
+
+This section describes the Agones-specific considerations in Autopilot
+clusters. For a general comparison between Autopilot and Standard, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
+
+Autopilot nodes are, by default, optimized for most workloads. If some of your
+workloads have broad compute requirements such as Arm architecture or a minimum
+CPU platform, you can also choose a
+[compute class](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes)
+that meets that requirement. However, if you have specialized hardware needs
+that require fine-grained control over machine configuration, consider using
+GKE Standard.
+
+Autopilot has pre-configured opinionated constraints. Evaluate whether these
+constraints impact your workloads:
```

Comment:
Maybe "Agones on Autopilot has pre-configured ..." since this is mixing both the Autopilot constraints and the Agones-on-Autopilot constraints?